City-of-Helsinki/berth-reservations-ui
# VEN-1105, VEN-1167 | Customer boats page. Customer page translations

## Description :sparkles:

* Customer boats page
* Customer page translations 🇸🇪 🇬🇧

## Issues :bug:

### Closes :no_good_woman:

* [VEN-1105](https://helsinkisolutionoffice.atlassian.net/browse/VEN-1105): Customer boats page
* [VEN-1167](https://helsinkisolutionoffice.atlassian.net/browse/VEN-1167): Add Swedish and English translations for new customer profile views

## Testing :alembic:

### Automated tests :gear:️

* None 😢 Let's add some when we add edit functionality here
* Included tests for formatWeight

### Manual testing :construction_worker_man:

* Go to the profile page `/profile/1` through `/profile/4` and check the boats tab. The contents are the same for all the "mock users".
